public/getting-started/index.html

```diff
@@ -152,7 +152,8 @@ <h2>Instructions</h2>
               <li><code>subtitle</code>: set this to a subtitle you want to show for
                 your site. It can be the empty string ("").</li>
               <li><code>doi</code>: set this to the empty string for now (""). (More
-                information on adding a DOI  is provided in the repository README.md file.)</li>
+                information on adding a DOI is provided in the
+                <a href="doi.html">DOI instructions</a>.)</li>
```

Comment:
I think this should be `"doi"` and the `doi.html` file should be moved to `public/getting-started/doi/index.html`. That way it's consistent with the other instructional pages (e.g. `/getting-started`, `examples/alpheios-integration/`) from a user's perspective.

Reply:
ok, will make that change too.